libtbx_refresh.py

```diff
@@ -29,6 +29,7 @@
 libtbx.pkg_utils.require('procrunner', '>=0.6')
 libtbx.pkg_utils.require('scipy')
 libtbx.pkg_utils.require('scikit_learn[alldeps]')
+libtbx.pkg_utils.require('tqdm', '==4.23.4')
```

Comment:
do we need to lock to a specific version?

Reply:
Yes - unfortunately the source package doesn't support turning on/off after a while, only being completely on or totally disabled. I've had to hook into private implementation-specific code, so obviously this has potential to change/break on even minor updates.